<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_stacked_widget</class>
 <widget class="QStackedWidget" name="search_stacked_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>513</height>
   </rect>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="loading_page">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="TimeoutProgressBar" name="timeout_progress_bar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>10</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="format">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="state_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="EllipseButton" name="show_results_button">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Show now</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="ChannelContentsWidget" name="results_page"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EllipseButton</class>
   <extends>QToolButton</extends>
   <header>tribler_gui.widgets.ellipsebutton.h</header>
  </customwidget>
  <customwidget>
   <class>ChannelContentsWidget</class>
   <extends>QWidget</extends>
   <header>tribler_gui.widgets.channelcontentswidget.h</header>
  </customwidget>
  <customwidget>
   <class>TimeoutProgressBar</class>
   <extends>QProgressBar</extends>
   <header>tribler_gui.widgets.timeoutprogressbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
